### PR TITLE
[4.0]Table text changed

### PR DESCRIPTION
**Pull request by puneeth-chanda**
Pull Request for Issue #22311

### Summary of Changes
'table' text changed to 'filter'

### CODE REVIEW

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -1051,4 +1051,5 @@ SQLITE="SQLite"
 
 ; Search tools
 JTABLE_OPTIONS="Table Options"
+JFILTER_OPTIONS="Filter Options"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.ini:1053]: Please sort in alpha order.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on b3ca96c1c5278650cc11fc0c3b3f7c8cdaba62df<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24022">issues.joomla.org/tracker/joomla-cms/24022</a>.</sub>

**Comment by puneeth-chanda** [comment]
@wilsonge can you please review this.

**Comment by ghost** [comment]
@puneeth2001 can you please append [4.0] in front of the Title.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**